cohere_sagemaker/embeddings.py

```diff
@@ -0,0 +1,27 @@
+from typing import Iterator, List
+
+from cohere.response import CohereObject
```

Comment:
it's not `cohere_sagemaker.response` here?

Reply:
Changed to use proper object